pkg/kubelet/kuberuntime/helpers_test.go

```diff
@@ -31,6 +31,50 @@ import (
 	utilpointer "k8s.io/utils/pointer"
 )
 
+func TestIsInitContainerFailed(t *testing.T) {
+	tests := []struct {
+		status   *kubecontainer.Status
+		isFailed bool
+	}{
+		{
+			status: &kubecontainer.Status{
+				State:    kubecontainer.ContainerStateExited,
+				ExitCode: 1,
+			},
+			isFailed: true,
+		},
+		{
+			status: &kubecontainer.Status{
+				State: kubecontainer.ContainerStateUnknown,
+			},
+			isFailed: true,
+		},
+		{
+			status: &kubecontainer.Status{
+				State:    kubecontainer.ContainerStateExited,
+				ExitCode: 0,
+			},
+			isFailed: false,
+		},
+		{
+			status: &kubecontainer.Status{
+				State: kubecontainer.ContainerStateRunning,
+			},
+			isFailed: false,
+		},
+		{
+			status: &kubecontainer.Status{
+				State: kubecontainer.ContainerStateCreated,
+			},
+			isFailed: false,
+		},
+	}
+	for i, test := range tests {
+		isFailed := isInitContainerFailed(test.status)
+		assert.Equal(t, test.isFailed, isFailed, "TestCase[%d]", i)
```

Comment:
Here should add the description, like
`assert.Equal(t, test.isFailed, isFailed, "TestCase[%d]: %s", i, t.description)`

Reply:
ok, I will add.